addons/event/static/src/client_action/event_barcode.js

```diff
@@ -116,27 +117,22 @@ export class EventScanView extends Component {
 
     onClickSelectAttendee() {
         if (this.isMultiEvent) {
-            this.actionService.doAction("event.event_registration_action");
+            this.actionService.doAction("event.event_registration_action", {
+                additionalContext: {
+                    is_from_registration_desk: true,
+                },
+            });
         } else {
             this.actionService.doAction("event.event_registration_action_kanban", {
                 additionalContext: {
                     active_id: this.eventId,
+                    is_from_registration_desk: true,
```

Comment:
actually is_registration_desk_view feels more comprehensive. Sorry for the back and forth ^^